ci/jenkins/README.md

```diff
@@ -0,0 +1,30 @@
+## Antrea CI: Jenkins
+
+### Reasons for Jenkins
+We have tests as Github Action but Jenkins allows tests running on a cluster of multiple nodes and offers better environment setup options.
+
+### List of Jenkins jobs
+* [e2e](https://github.com/vmware-tanzu/antrea/tree/master/test/e2e): end-to-end tests for Antrea.
+* conformance: community tests using sonobuoy, focusing on "Conformance" and skipping "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli", "sig-storage", "sig-auth", "sig-api-machinery", "sig-apps", or "sig-node".
+* network policy: community tests using sonobuoy, focusing on "Feature:NetworkPolicy" and skipping "allow ingress access from updated pod" or "named port".  
+Test "named port" will be supported soon, issue [#122](https://github.com/vmware-tanzu/antrea/issues/122).  
+Test "allow ingress access from updated pod" is a bug to be fixed, issue [#85908](https://github.com/kubernetes/kubernetes/issues/85908).
```

Comment:
```suggestion
Test "allow ingress access from updated pod" fails because of a bug in the test definition, see issue [#85908]
```

Reply:
I am not sure why you delete the link to kubernetes issue but keep the "[]" around "#85908" ?

Reply:
I think that's just a typo on my part or something weird with the Github UI.

Reply:
@lzhecheng I think you should put the link back, it was a typo on my part. I did not copy the whole line when I created the suggestion.

Reply:
OK, I have put back the link.